### PR TITLE
K8s 1.26 release updates

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- Added 1.26 release notes
- Added 1.26 components
- Added 1.26 upgrade instructions
- Update equinix overlay
- add clarification for vSphere
- remove warnings which no longer apply
- update metallb instructions

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by anthonydillon** [comment]
@anthonydillon let me know when this is ready for review.

**Comment by evilnick** [comment]
@anthonydillon hi! sorry for the delay, waiting for the last glitches to get ironed out.

File: templates/kubernetes/docs/1.25/components.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Other information about this release can be found on the following pages:
 ## What's new
 
 For a list of new features, changes, deprecations, and bug fixes in this
-release, please see the [release notes](release-notes).
+release, please see the [Release notes](release-notes).
```
Comment (anthonydillon) [inline templates/kubernetes/docs/1.25/components.md:40]: Questionable capitalisation?

File: templates/kubernetes/docs/1.25/components.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ release. These charms are maintained by the Charmed Kubernetes team.
 
 
 ## CNI charms
-These are the CNI charms also supported by Charmed Kubernetes (calico is the default CNI and is included in the core charms above)
+These are the CNI charms also supported by Charmed Kubernetes ( calico is the default CNI and is included in the core charms above)
```
Comment (anthonydillon) [inline templates/kubernetes/docs/1.25/components.md:60]: Why add a space (nit)

**Review by anthonydillon — APPROVED** [review]
Huge amount of changes

File: templates/kubernetes/docs/1.25/components.md (hunk 1 above)
Comment (evilnick) [reply templates/kubernetes/docs/1.25/components.md:40]: Yes, this seems to be a regression, will address it across the source next pulse

**Comment by evilnick** [comment]
@anthonydillon Have made those small tweaks, please feel free to merge( I can't) when ready if you are happy

**Pull request merged**